SDL/Quintuplet.cu

```diff
@@ -163,6 +185,14 @@ cudaStreamSynchronize(stream);
     }
     cudaMemcpyAsync(rangesInGPU.quintupletModuleIndices,module_quintupletModuleIndices,nLowerModules*sizeof(int),cudaMemcpyHostToDevice,stream);
     cudaMemcpyAsync(rangesInGPU.nEligibleT5Modules,&nEligibleModules,sizeof(uint16_t),cudaMemcpyHostToDevice,stream);
+//#ifdef CACHE_ALLOC
+//        int dev;
+//        cudaGetDevice(&dev);
+//        rangesInGPU.indicesOfEligibleT5Modules = (uint16_t*)cms::cuda::allocate_device(dev, nEligibleModules * sizeof(uint16_t), stream);
+//#else
+//        cudaMalloc(&(rangesInGPU.indicesOfEligibleT5Modules), nEligibleModules * sizeof(uint16_t));
+//#endif
```

Comment:
I can delete it if you think it is a good idea. The reason I left it was because this was in the old cpu function and I wanted all the parts to be there in case we need to go back. But I can actually just delete the entire function since it is not used anymore

Reply:
Sure, if these lines need to be commented in in case we want to go back, it makes sense for them to be here. I just thought that they were moved to the `Event.cu` file.

Reply:
Yes, the allocations was moved to Event.cu. so one should be able to run this function with that allocation done in Event instead of inside this function. but if you wanted to go back completely, you would run this function with the lines uncommented and then comment out (or remove) the lines in event.cu. If i remove these commeted out lines, then if i would try to go back later, I might not remember this and run with a slightly different setup (with the lines in event still there). Probably (assuming we accept these changes), I should just delete the old function entirely.